src/Common/src/Interop/Windows/sspicli/SecuritySafeHandles.cs

```diff
@@ -872,7 +872,7 @@ public new IntPtr DangerousGetHandle()
                     GlobalLog.Assert("SafeDeleteContext::AcceptSecurityContext()|inSecBuffer == null || inSecBuffers == null");
                 }
 
-                Debug.Fail("SafeDeleteContext::AcceptSecurityContext()|outSecBuffer != null");
+                Debug.Fail("SafeDeleteContext::AcceptSecurityContext()|inSecBuffer == null || inSecBuffers == null");
```

Comment:
I was trying to follow the pattern used elsewhere; other places state what's expected rather than what was received, so I did that here, too (copied from above).